packages/cli-app/src/handlers/register.ts

```diff
@@ -89,27 +89,24 @@ export const register = (config: any, { getAppRecord, getDXNSClient }: RegisterP
     assert(/^[a-zA-Z0-9][a-zA-Z0-9-.]{1,61}[a-zA-Z0-9-]{2,}$/.test(name), 'Name could contain only letters, numbers, dashes or dots.');
 
     // TODO(egorgripasov): Adapter for the new record format. Cleanup.
-    const { name: appName, version, author, description, package: pkg, ...rest } = conf;
+    const { version, description, package: pkg, ...rest } = conf;
 
-    const client: { registryApi: IRegistryApi } = await getDXNSClient();
+    const client: { registryClient: IRegistryClient } = await getDXNSClient();
 
-    const appType = await client.registryApi.getResource<RegistryTypeRecord>(DXN.parse(APP_DXN_NAME));
+    const appType = await client.registryClient.getResource<RegistryTypeRecord>(DXN.parse(APP_DXN_NAME));
     assert(appType);
     assert(appType.record.kind === RecordKind.Type);
 
-    const cid = await client.registryApi.insertDataRecord({
+    const cid = await client.registryClient.insertDataRecord({
       hash: CID.from(pkg['/']).value,
       ...rest
     }, appType?.record.cid, {
-      created: new Date(),
       version,
-      author,
-      description,
-      name: appName
```

Comment:
@rzadp just to confirm - do we want to remove name from apps/bots (which is actually DisplayName)?

Reply:
@egorgripasov Yes, we want to remove the name here - the name comes from a resource now, and the resources will be listed with display names, not records.